_data/toc/page-builder.yml

```diff
@@ -2,7 +2,7 @@ label: Page Builder
 versionless: true
 pages:
 - label: What is Page Builder?
-  url: /page-builder/docs/
+  url: /page-builder/docs/index.html
```

Comment:
Should this be extensionless, like line 149?

Reply:
It's fine as is but I'll make it consistent.